### ISSUE TITLE
[Bug] File upload - Network error

### ISSUE THREAD

**Issue by annaswims** [department-of-veterans-affairs/va.gov-team#14427]
## Summary
In both the Claim Status tool and Form526 form we see  `net::ERR_FAILED` when we try to upload [this file](https://github.com/department-of-veterans-affairs/vagov-content/blob/master/assets/playbook/downloads/VHA_Style_Guide_508.pdf)

## Details
Logged in as user 229, I tried to upload the file in either the Claim status tool or Form 526:

web url: https://staging.va.gov/track-claims/your-claims/600207281/files
api endpoint: https://staging-api.va.gov/v0/evss_claims/600207281/documents
![Screen Shot 2020-10-05 at 5 23 35 PM](https://user-images.githubusercontent.com/19188/95134755-6ab57180-0731-11eb-9eb5-4c538990cd89.png)

web url:  https://staging.va.gov/disability/file-disability-claim-form-21-526ez/review-and-submit
api endpoint: https://staging-api.va.gov/v0/upload_supporting_evidence
![Screen Shot 2020-10-05 at 4 58 25 PM](https://user-images.githubusercontent.com/19188/95134753-69844480-0731-11eb-8baf-f9e57a145803.png)

## Impact
Within forms,  6.5k users have encountered this issue 14k times. 
[old sentry](http://sentry.vfs.va.gov/vets-gov/website-production/issues/14738/?query=Network%20request%20failed)
[New Sentry](http://sentry10.vfs.va.gov/organizations/vsp/issues/144/?project=4&query=is%3Aunresolved)
within the claim status tool we don't know how many people were affected. 

## Related:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/14425 - Log upload errors to Sentry. 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/13293 - Even when the file makes it, it may be messed up later.

**Comment by annaswims** [comment]
you can ignore the bit in the screenshots where sentry failed. That was a [separate issue](https://dsva.slack.com/archives/CBU0KDSB1/p1601930770132700)

**Comment by annaswims** [comment]
In some [lighthouse docs](https://developer.va.gov/explore/benefits/docs/claims?version=current) they talk about Base64 encoding files, I think this a workaround for TIC issues.

**Comment by annaswims** [comment]
We should talk to @bastosmichael about what lead him to do this https://github.com/department-of-veterans-affairs/vets-api/pull/4227/files It's likely trying to solve the same issue.

**Comment by annaswims** [comment]
I'm hopeful that this might help https://github.com/department-of-veterans-affairs/devops/pull/7725

**Comment by annaswims** [comment]
😱 😱 😱 😱 😱 
When I send the same request via postman I get a 413 Request Entity Too Large response, but when I send the same request from vets-website I get:
`Access to XMLHttpRequest at 'https://api.va.gov/v0/evss_claims/80390321/documents' from origin 'https://www.va.gov' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`
[
<img width="1516" alt="Screen Shot 2020-10-06 at 9 33 05 AM" src="https://user-images.githubusercontent.com/19188/95208402-09d07c80-07b7-11eb-8e37-c2e40bc61849.png">
](url)


So... what's happening is that when nginx isn't setting `Access-Control-Allow-Origin`, which means that the response isn't available to vets-website.  

We really need a way to make the response available to vets-website.  I think that people triaging the sentry issue incorrectly assumed that the `vets_upload_error: Network request failed` errors we were seeing in sentry were related to unstable networks, which is not the case.

**Comment by luke-gcio** [comment]
Is this not within our (BAM1) control?

**Comment by annaswims** [comment]
https://github.com/department-of-veterans-affairs/devops/pull/7725 increased the request limit to 50mb so we'll now need a >50mb file to reproduce. I talked to ops during open office hours and they said they'll look at the headers in the revproxy.

**Comment by annaswims** [comment]
tl;dr we should set `Access-Control-Allow-Origin` in nginx to [these values](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2#L6) if the headers aren't set by rails

**Comment by luke-gcio** [comment]
Note to Luke from Luke: Get this assigned to the right people/team.  Devops tag already there.

**Comment by luke-gcio** [comment]
New ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/15687
